### ISSUE TITLE
upgrade from 1.13.0 => 1.13.2 on Mac OS X massively breaks linting

### ISSUE THREAD

**Issue by jdavidheiser** [AtomLinter/linter-flake8#212]
When I upgraded to 1.13.2, linting almost never happened, and seemed to only be triggered when I had a really egregious syntax error.  Downgrading back to 1.13.0 fixed the problem.

The behavior seems very similar to that in #175

**Comment by Arcanemagus** [comment]
The original reporter of that issue stopped responding, and the other person commenting there traced it down to a problematic plugin to `flake8`.

What do you get for `flake8 --version`?

**Comment by jdavidheiser** [comment]
I get:

`2.6.2 (pycodestyle: 2.0.0, pyflakes: 1.2.3, flake8_quotes: 0.3.0, mccabe: 0.5.0) CPython 2.7.11 on Darwin`

**Comment by Arcanemagus** [comment]
If you (temporarily) remove `flake8_quotes` does it work?

**Comment by jdavidheiser** [comment]
By following the tip in that other issue, I found that flake8_quotes was not working correctly when called with stdin, and fixing that bug seems to have resolved this, although I was changing many other things at the same time, so it could have been something else.  Basically, a complete uninstall of both the python 2 and python 3 version of flake8, plus nuking my .atom directory, and fixing the bug in flake8_quotes seems to have done the trick.

It was weird that downgrading back to 1.13.0 ALSO fixed the problem, but after all of the other changes I was able to upgrade back to the latest version and it still worked.
